Add Device Icons for WebOS & Updated Infuse Naming

JPKribs opened the pull request:
**Changes**
First time doing anything on Web so please let me know if this isn't right! 

[Over in Swiftfin](https://github.com/jellyfin/Swiftfin/pull/1275), I added Device Logos and I threw in a webOS (LG TVs) icon as well. I just wanted to add that logo and the logic so Jellyfin-Web had it as well. In my testing, 'LG Smart TV' covers all of the TVs (all 2 I had to test) but I'm unsure if there is a different name for projectors?

I am using webOS's icon from here: https://en.wikipedia.org/wiki/WebOS#/media/File:LG_WebOS_New.svg.

Additionally, during my Swiftfin changes, I found that Infuse doesn't use 'Infuse' as a device name after 7.7(?) and now uses 'Infuse-Direct' or 'Infuse-Library' on later versions. While I was adding WebOS, I added this change as well. I left the original just plain 'Infuse' for anyone running previous versions as well.

**Issues**
https://github.com/jellyfin/jellyfin-web/issues/6280

viown commented:
I get this when I try the icon:

![image](https://github.com/user-attachments/assets/493070b2-c4a9-49fe-a940-aba37cb68ac3)

JPKribs commented:
> I get this when I try the icon:
> 
> ![image](https://private-user-images.githubusercontent.com/48097677/382135767-493070b2-c4a9-49fe-a940-aba37cb68ac3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1MjU2ODUsIm5iZiI6MTczMDUyNTM4NSwicGF0aCI6Ii80ODA5NzY3Ny8zODIxMzU3NjctNDkzMDcwYjItYzRhOS00OWZlLWE5NDAtYWJhMzdjYjY4YWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAyVDA1Mjk0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVjM2Y4MGZlMTM1NWFjNTJlMWQzNzM2NTkxODQ1N2QzY2U3YTNlY2Q5MmQ0ZjMyNzc3MzlkYzFmMDE1NTdlYmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pWA-xGpLs_RxuleBC2OtIFXryr6N2vwUVs-YgljD5nU)

I'm sorry about that! I think I have this resolved. Additionally, while I was looking at my devices I realized Swiftfin wasn't in there on web for Apple either! So I've added both variants of Swiftfin. It looks like 1.0.1 and below is just Jellyfin tvOS/iOS/iPadOS and 1.1+ is Swiftfin tvOS/iOS/iPadOS. Additionally, I put the client name in for LG TV since it looks like we're looking at both client and device.

Let me know if this scaling looks right! I added some items next to it for comparison:

![Untitled](https://github.com/user-attachments/assets/6712c451-a8a9-42d3-87cc-7ddfe971fdc6)

JPKribs commented:
Oh! Lastly, the Samsung.svg should be Samsungtv.svg. I updated that too.

thornbill commented:
I don't think we should add padding to the logo image to fix what is really a bug with the css. (I also don't think that fixing that bug is a blocker for this PR. :slightly_smiling_face:)

JPKribs commented:
> I don't think we should add padding to the logo image to fix what is really a bug with the css. (I also don't think that fixing that bug is a blocker for this PR. 🙂)

Sounds good! I think on my last iteration the WebOS logo should match the same size as the other logos. Would you rather I trim that to just be the size of the actual logo?

Let me know what works best! Aside from CSS changes, that might be a little beyond me haha.

thornbill commented:
Yeah I would just make it the size of the logo itself. I had a fix for the css the other day but I think I deleted it at some point. I'll open a PR for it when I get it fixed again. 🙃

JPKribs commented:
There we go! I kept updating the image but GitHub was showing it with huge vertical margins. I think that might be just how GitHub displays svgs? Either way, I've updated the image so it should just be exactly their [Wikipedia SVG](https://upload.wikimedia.org/wikipedia/commons/5/59/LG_WebOS_New.svg) but white to mirror our existing logos.

I'll also directly attach the svg to this comment just in case something weird is going on.

![webos.svg](https://github.com/user-attachments/assets/52eb41d3-d1ef-425c-b0e6-0ae74bb143a4)
